test/requirements-test.txt

```diff
@@ -16,6 +16,7 @@ git+https://github.com/microsoft/TransformerCompression.git ; python_version >=
 mlflow>=2.4.0
 neural-compressor
 nncf==2.7.0
+nvidia-modelopt
```

Comment:
Looks like onnx-graphsurgeon is also required. From the windows ci failure:
![image](https://github.com/microsoft/Olive/assets/94929125/06418d6e-5030-4561-aed6-5d2bb1fb47bf)

Reply:
I have updated the requirement with correct package.
qq: we have a dependency on `onnxruntime~=1.16.3`, will that conflict with onnxruntime requirement for other packages/tools within olive?

Reply:
Oh, that is an issue. Our CI tests for the most recent stable release of ort. We have another pipeline that tests for the previous major release of ort but that one will soon migrate to 1.17.x since 1.18 is on the way.

Reply:
How to see the status of the pipeline? The `int4` support that we added in this PR does not require strict `1.16.3`, it will work with latest ort too. Let see if pipeline passes.
We will update our ort dependency to latest version for other quantization formats like `fp8`, `int8` before supporting them in olive.

Reply:
okay, let me trigger the ci pipeline. you can see the status by clicking the jobs that show up. it takes you to another page that has links to the azure pipeline jobs.

Reply:
I see the the `[all]` extra pins the ort version so this will reinstall ort. Can we directly install onnx-graphsurgeon package so that the ort package is not reinstalled? 
![image](https://github.com/microsoft/Olive/assets/94929125/cbd0f939-5b5d-408f-9918-136d0aac39ed)

Reply:
I think it is doable, I have updated the requirements.
Bdw, I do not see `onnxruntime` as dependency in requirements-tests.txt, is it coming through core lib requirements?

Reply:
it is installed separately so that we can control the version being tested  https://github.com/microsoft/Olive/blob/63c0430fe3da033e086e1dfbe00d9bdc918ecca8/.azure_pipelines/job_templates/olive-test-template.yaml#L37